--- atk-1.3.2/configure.in.orig	Thu May 15 07:30:58 2003
+++ atk-1.3.2/configure.in	Mon May 19 07:47:07 2003
@@ -86,7 +86,7 @@
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, ["$GETTEXT_PACKAGE"],
   [Define the gettext package to be used])
 
-ALL_LINGUAS="am az be bg bn ca cs da de en_GB el eo es fa fi fr he hu kn ko id is it ja lv ml mk mn ms nl no pl pt pt_BR ro ru sk sq sr sr@Latn sv tr uk vi yi zh_CN zh_TW"
+ALL_LINGUAS="am az be bg bn ca cs da de en_GB el eo es fa fi fr he hu kn ko id is it ja lv ml mk mn ms nl no pl pt pt_BR ro ru sk sq sr sr@cyrillic sv tr uk vi yi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 GLIB_DEFINE_LOCALEDIR(atklocaledir)
