diff -Nru atk-1.1.0.orig/docs/Makefile.am atk-1.1.0/docs/Makefile.am
--- atk-1.1.0.orig/docs/Makefile.am	Fri Aug 16 17:36:55 2002
+++ atk-1.1.0/docs/Makefile.am	Wed Oct  9 15:06:12 2002
@@ -119,9 +119,20 @@
 sgml.stamp: sgml-build.stamp
 	@true
 
+#### xml ####
+
+xml-build.stamp: tmpl.stamp $(CFILE_GLOB) $(srcdir)/tmpl/*.sgml
+	@echo '*** Building XML ***'
+	cd $(srcdir) && \
+	gtkdoc-mkdb --module=$(DOC_MODULE) --source-dir=$(DOC_SOURCE_DIR) --output-format=xml $(MKDB_OPTIONS)
+	touch xml-build.stamp
+
+xml.stamp: xml-build.stamp
+	@true
+
 #### html ####
 
-html-build.stamp: sgml.stamp $(DOC_MAIN_SGML_FILE) $(content_files)
+html-build.stamp: xml.stamp $(DOC_MAIN_SGML_FILE) $(content_files)
 	@echo '*** Building HTML ***'
 	rm -rf $(srcdir)/html 
 	mkdir $(srcdir)/html
diff -Nru atk-1.1.0.orig/docs/atk-docs.sgml atk-1.1.0/docs/atk-docs.sgml
--- atk-1.1.0.orig/docs/atk-docs.sgml	Thu Jun 20 10:29:32 2002
+++ atk-1.1.0/docs/atk-docs.sgml	Wed Oct  9 14:59:30 2002
@@ -1,27 +1,29 @@
-<!doctype book PUBLIC "-//Davenport//DTD DocBook V3.0//EN" [
-<!entity atk-AtkAction SYSTEM "sgml/atkaction.sgml">
-<!entity atk-AtkComponent SYSTEM "sgml/atkcomponent.sgml">
-<!entity atk-AtkDocument SYSTEM "sgml/atkdocument.sgml">
-<!entity atk-AtkEditabletext SYSTEM "sgml/atkeditabletext.sgml">
-<!entity atk-AtkGObjectAccessible SYSTEM "sgml/atkgobjectaccessible.sgml">
-<!entity atk-AtkHyperlink SYSTEM "sgml/atkhyperlink.sgml">
-<!entity atk-AtkHypertext SYSTEM "sgml/atkhypertext.sgml">
-<!entity atk-AtkImage SYSTEM "sgml/atkimage.sgml">
-<!entity atk-AtkNoOpObject SYSTEM "sgml/atknoopobject.sgml">
-<!entity atk-AtkNoOpObjectFactory SYSTEM "sgml/atknoopobjectfactory.sgml">
-<!entity atk-AtkObject SYSTEM "sgml/atkobject.sgml">
-<!entity atk-AtkObjectFactory SYSTEM "sgml/atkobjectfactory.sgml">
-<!entity atk-AtkRegistry SYSTEM "sgml/atkregistry.sgml">
-<!entity atk-AtkRelation SYSTEM "sgml/atkrelation.sgml">
-<!entity atk-AtkRelationSet SYSTEM "sgml/atkrelationset.sgml">
-<!entity atk-AtkSelection SYSTEM "sgml/atkselection.sgml">
-<!entity atk-AtkState SYSTEM "sgml/atkstate.sgml">
-<!entity atk-AtkStateSet SYSTEM "sgml/atkstateset.sgml">
-<!entity atk-AtkStreamableContent SYSTEM "sgml/atkstreamablecontent.sgml">
-<!entity atk-AtkTable SYSTEM "sgml/atktable.sgml">
-<!entity atk-AtkText SYSTEM "sgml/atktext.sgml">
-<!entity atk-AtkUtil SYSTEM "sgml/atkutil.sgml">
-<!entity atk-AtkValue SYSTEM "sgml/atkvalue.sgml">
+<?xml version="1.0"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" 
+               "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" [
+<!ENTITY atk-AtkAction SYSTEM "xml/atkaction.xml">
+<!ENTITY atk-AtkComponent SYSTEM "xml/atkcomponent.xml">
+<!ENTITY atk-AtkDocument SYSTEM "xml/atkdocument.xml">
+<!ENTITY atk-AtkEditabletext SYSTEM "xml/atkeditabletext.xml">
+<!ENTITY atk-AtkGObjectAccessible SYSTEM "xml/atkgobjectaccessible.xml">
+<!ENTITY atk-AtkHyperlink SYSTEM "xml/atkhyperlink.xml">
+<!ENTITY atk-AtkHypertext SYSTEM "xml/atkhypertext.xml">
+<!ENTITY atk-AtkImage SYSTEM "xml/atkimage.xml">
+<!ENTITY atk-AtkNoOpObject SYSTEM "xml/atknoopobject.xml">
+<!ENTITY atk-AtkNoOpObjectFactory SYSTEM "xml/atknoopobjectfactory.xml">
+<!ENTITY atk-AtkObject SYSTEM "xml/atkobject.xml">
+<!ENTITY atk-AtkObjectFactory SYSTEM "xml/atkobjectfactory.xml">
+<!ENTITY atk-AtkRegistry SYSTEM "xml/atkregistry.xml">
+<!ENTITY atk-AtkRelation SYSTEM "xml/atkrelation.xml">
+<!ENTITY atk-AtkRelationSet SYSTEM "xml/atkrelationset.xml">
+<!ENTITY atk-AtkSelection SYSTEM "xml/atkselection.xml">
+<!ENTITY atk-AtkState SYSTEM "xml/atkstate.xml">
+<!ENTITY atk-AtkStateSet SYSTEM "xml/atkstateset.xml">
+<!ENTITY atk-AtkStreamableContent SYSTEM "xml/atkstreamablecontent.xml">
+<!ENTITY atk-AtkTable SYSTEM "xml/atktable.xml">
+<!ENTITY atk-AtkText SYSTEM "xml/atktext.xml">
+<!ENTITY atk-AtkUtil SYSTEM "xml/atkutil.xml">
+<!ENTITY atk-AtkValue SYSTEM "xml/atkvalue.xml">
 ]>
 
 <book>
